MonoGame.Framework/Graphics/SpriteBatcher.cs

```diff
@@ -224,7 +224,16 @@ public void DrawBatch(SpriteSortMode sortMode, Effect effect)
 
             // Determine how many iterations through the drawing code we need to make
             int batchIndex = 0;
-            int batchCount = _batchItemList.Count;
+            int batchCount = _batchItemList.Count;
+
+            if (_device._enableMetrics)
+            {
+                unchecked
+                {
+                    _device._metrics._batchCount += (ulong)batchCount;
```

Comment:
I think the terminology here is weird....  the `batchCount` here really means sprite count.  You are really counting number of sprites being rendered (with text being one sprite per-character).

Is this what we really want to convey?  Should it be `Metrics.SpriteCount`?